clang/lib/Sema/SemaOverload.cpp

```diff
@@ -11758,6 +11795,40 @@ static void DiagnoseBadDeduction(Sema &S, NamedDecl *Found, Decl *Templated,
       return;
     }
 
+    // Errors in deduction guides from inherited constructors
+    // will present as substitution failures in the mapping
```

Comment:
```suggestion
    // will manifest as substitution failures in the corresponding
```

(not 100% sure what you mean by mapping?)

Reply:
I've updated the comment. `Mapping` here refers to the partial specialization that maps the return type of the original deduction guide back to the derived class